### ISSUE TITLE
Erroneous behavior when running `sudo /this/file/does/not/exist`

### ISSUE THREAD

**Issue by marmarta** [sudo-project/sudo#193]
Distro: Fedora 36, updated from Fedora 35

In `sudo 1.9.7p2` running a non-existing command for a user in the `wheel` group resulted in:

```
[user ~]$ sudo /this/file/does/not/exist
sudo: /this/file/does/not/exist: command not found
```

In version `1.9.8p2` I get:

```
[user ~]$ sudo /this/file/does/not/exist

We trust you have received the usual lecture from the local System
Administrator. It usually boils down to these three things:

    #1) Respect the privacy of others.
    #2) Think before you type.
    #3) With great power comes great responsibility.

[sudo] password for user: 
```

Which is somewhat problematic if there is, e.g., a script that relied on command erroring-out if it does not exist for some reason.

**Comment by millert** [comment]
I believe this was fixed in sudo 1.9.12 by 42117a1ce22745eb07eb1cff193892fe511dbff5.

**Comment by millert** [comment]
Verified that this was fixed by 42117a1ce22745eb07eb1cff193892fe511dbff5 in sudo 1.9.11.
